src/OutputWriters/output_writer_utils.jl

```diff
@@ -27,7 +28,7 @@ saveproperty!(file, location, p::Tuple) =
 saveproperty!(file, location, p) =
     [saveproperty!(file, location * "/$subp", getproperty(p, subp)) for subp in propertynames(p)]
 
-saveproperty!(file, location, p::ImmersedBoundaryGrid) = saveproperty!(file, location, p.grid)
+saveproperty!(file, location, p::ImmersedBoundaryGrid) = saveproperty!(file, location, grid)
```

Comment:
I don't understand where from the `saveproperty!` knows `grid`.

Reply:
Thats a typo / bug.